build.cmd

```diff
@@ -481,6 +481,9 @@ if NOT EXIST Proto\net40\bin\fsc.exe (
   set BUILD_PROTO=1
 )
 
+rem turn off vs ide unit tests until they pass again
+set TEST_VS_IDEUNIT_SUITE= 0
```

Comment:
Note: 
now that the build is fixed and the references match, the tests can run again.  however they have bit rot.  the test fix will follow.


disable vside tests until the tests are fixed.

Reply:
@KevinRansom given that this will be very time consuming, can we yank out the Roslyn-based unit tests from the legacy ones and enable them? They have not bit-rot and should be running to prevent further regressions.